src/Nethermind/Nethermind.Logging/MicrosoftLoggerExtensions.cs

```diff
@@ -0,0 +1,66 @@
+//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using Microsoft.Extensions.Logging;
+
+namespace Nethermind.Logging
```

Comment:
These have very little in common with your previous Nethermind.Logging project.

I would suggest, if you do add extensions to the standard Microsoft.Extensions.Logging, that they are a in different library, as you will either use one or the other, never mix both.

Reply:
Yeah, I know what I do - same thing as with NLog - let me fix it